src/utils/index.ts

```diff
@@ -7,6 +7,9 @@ import { InjectedConnector } from '@web3-react/injected-connector'
 import { WalletConnectConnector } from '@web3-react/walletconnect-connector'
 import { NetworkConnector } from '@web3-react/network-connector'
 
+export const isProduction = (): boolean =>
+  !!process.env.CONTEXT && process.env.CONTEXT == 'production'
```

Comment:
Just note, the default environment variable for this is: `NODE_ENV` for most next.js app. All the build/deploy tools should pick this up by default.